pynicotine/gtkgui/ui/dialogs/searchfilters.ui

```diff
@@ -2,7 +2,7 @@
 <interface>
   <requires lib="gtk+" version="3.18"/>
   <object class="GtkDialog" id="AboutSearchFiltersDialog">
-    <property name="title" translatable="yes">Search Filters</property>
+    <property name="title" translatable="yes">Result Filters</property>
```

Comment:
I feel naming this "Search Result Filters" would make it easier to understand the context at a quick glance, mainly in the main menus.

Reply:
I was thinking of this too, but though it may be too long. But I'm okay with this compromise.

Reply:
It's roughly the same length as the other help menu items, so it shouldn't be an issue.